src/lib/viewers/__tests__/BaseViewer-test.js

```diff
@@ -1884,5 +1890,26 @@ describe('lib/viewers/BaseViewer', () => {
             base.handleAnnotationControlsClick({ mode: AnnotationMode.NONE });
             expect(base.annotator.toggleAnnotationMode).to.be.calledWith(AnnotationMode.REGION);
         });
+
+        it('should add create region class if discoverability is enabled and mode is REGION', () => {
+            base.options.enableAnnotationsDiscoverability = true;
+            base.handleAnnotationControlsClick({ mode: AnnotationMode.REGION });
+
+            expect(base.containerEl).to.have.class(CSS_ANNOTATIONS_CREATE_REGION);
+        });
+
+        it('should remove create region class if discoverability is enabled and mode is NONE', () => {
+            base.options.enableAnnotationsDiscoverability = true;
+            base.handleAnnotationControlsClick({ mode: AnnotationMode.NONE });
+
+            expect(base.containerEl).to.not.have.class(CSS_ANNOTATIONS_CREATE_REGION);
+        });
+
+        it('should remove create region class if discoverability is enabled and mode is HIGHLIGHT', () => {
+            base.options.enableAnnotationsDiscoverability = true;
+            base.handleAnnotationControlsClick({ mode: AnnotationMode.HIGHLIGHT });
+
+            expect(base.containerEl).to.not.have.class(CSS_ANNOTATIONS_CREATE_REGION);
```

Comment:
Could combine these in a parameterized test?

Reply:
Is this how it would look?

```
const parameters = [
            { description: 'should remove create region class if discoverability is enabled and mode is NONE', mode: AnnotationMode.NONE },
            { description: 'should remove create region class if discoverability is enabled and mode is HIGHLIGHT', mode: AnnotationMode.HIGLIGHT },
        ];

        parameters.forEach((parameter) => {
            it(parameter.description, () => {
                base.options.enableAnnotationsDiscoverability = true;
                base.handleAnnotationControlsClick({ mode: parameter.mode });
    
                expect(base.containerEl).to.not.have.class(CLASS_ANNOTATIONS_CREATE_REGION);
            });
        });
```

Reply:
Probably could be simpler like:
```
[AnnotationMode.NONE, AnnotationMode.HIGHLIGHT].forEach(mode => {
  it(`should remove create region class if discoverability is enabled and mode is ${mode}`, () => { ... })
});
```